### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Ash-D23**
- Added Home Page
- Display categories
-  Display featured product
- Added loader when async call happens

sc1
sc2

### CODE REVIEW

**Review by Pratik1005 — APPROVED** [review]
At few places there is class instead of className. Rest looks good

File: src/Pages/HomePage/HomePage.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#shop{
+    background: #8e9eab;
+    background: -webkit-linear-gradient(to bottom, #eef2f3, #8e9eab);
+    background: linear-gradient(to bottom, #eef2f3, #8e9eab);
```
Comment (manismk) [inline src/Pages/HomePage/HomePage.css:4]: If possible try to use cass variables here

File: src/Pages/HomePage/HomePage.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+.category--card img{
+    opacity: 0.9;
+} 
+
+.shadow--bottom{
+    -moz-box-shadow: inset 0 -60px 60px -60px #000000;
+    -webkit-box-shadow: inset 0 -60px 60px -60px #000000;
+    box-shadow: inset 0 -60px 60px -60px #000000;
```
Comment (manismk) [inline src/Pages/HomePage/HomePage.css:45]: Here too

**Review by manismk — APPROVED** [review]
At most of the places instead className its given as classNameName.
Rest looks good. Good folder structure and good naming convention.

File: src/Pages/HomePage/HomePage.css (hunk 2 above)
Comment (Ash-D23) [reply src/Pages/HomePage/HomePage.css:45]: its a one time usecase, gradient colors hence not added them as variables in root

**Pull request merged**